internal/service/common/utils/utils.go

```diff
@@ -7,6 +7,10 @@ import (
 )
 
 type DBTag map[string]string
+type DBValue map[string]interface{}
+
+const IncludeNilValues = false
```

Comment:
these are confusing, better to create a specific function

Reply:
restructured this a bit so still reuse a common function but to not expose these options.